tradeexecutor/statistics/core.py

```diff
@@ -46,7 +46,6 @@ def calculate_statistics(clock: datetime.datetime, portfolio: Portfolio) -> NewS
     """Calculate statistics for a portfolio."""
 
     first_trade, last_trade = portfolio.get_first_and_last_executed_trade()
-    trade_analysis = build_trade_analysis(portfolio)
 
     pf_stats = PortfolioStatistics(
```

Comment:
You can check `ExecutionMode` if we are in backtesting. In backtesing we can skip this, but in live trading we want to see the stats in the frontend after each cycle.